src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Helpers/Parser/CandidateBinderInvocation.cs

```diff
@@ -8,12 +8,12 @@
 
 namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration
 {
-    internal sealed record BinderInvocation
+    internal sealed record CandidateBinderInvocation
```

Comment:
This type is `record` but is not actually value equals due to the use of `IInvocationOperation` whihc is not equatable.